configure.cmd

```diff
@@ -151,7 +135,6 @@ REM Parse command line parameters
                     set CMAKE_GENERATOR="Visual Studio 11 Win64"
                 ) else if "!ARCH!" == "arm" (
                     set CMAKE_GENERATOR="Visual Studio 11 ARM"
-                    set CMAKE_GENERATOR_HOST="Visual Studio 11"
```

Comment:
Why are these removed?
In some form or way, the host tools should be built with the 'native' toolset, not with the `ARM` toolset

Reply:
The only place where it's used is being removed...
https://github.com/reactos/reactos/pull/1356/files#diff-3ac55282d165a792e56e4f7af827fd90L266

Reply:
so this, in turn should also not be removed?
Or get an alternative code path.

Reply:
It doesn't seem to work as intended, or at least i couldn't get it to work, the replacement is configuring and building host tools separately and using `-DWITH_HOST_TOOLS=path`

Reply:
`This is the only occurrence that I could find of `WITH-HOST-TOOLS`:
https://git.reactos.org/?p=reactos.git;a=blob;f=configure.sh;hb=a976ebb0315a0b27438edb04fd69555e7de732df#l37

Reply:
its a cmake argument, https://github.com/reactos/reactos/blob/6ad548a49153bd28e47b2e6d30543faa6a0b7ef2/sdk/cmake/host-tools.cmake#L48-L58

Reply:
My bad, I checked in master instead of in your pr.